docs/components/base/boat.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Configure a Boat Base"
+linkTitle: "boat"
+weight: 35
+type: "docs"
+draft: "true"
+description: "Configure a boat base."
+tags: ["base", "components"]
+# SMEs: Eliot
+---
+
+A `boat` base is a model for a mobile robotic boat.
```

Comment:
I think more information is needed here because it seems we're assuming there's a right and left motor? What kind of boat?

Reply:
I don't know, this is draft